client/app/host/page.tsx

```diff
@@ -17,12 +15,11 @@ import {
     DropdownMenuSubTrigger,
     DropdownMenuTrigger,
 } from '@/components/ui/dropdown-menu';
-import { Ellipsis, Moon, Sun, SunMoon, Copy } from 'lucide-react';
-import HeaderMenu from './HeaderMenu';
+import { Ellipsis, Copy } from 'lucide-react';
 import Timer from '@/components/Timer';
+import HostNavbar from './HostNavbar';
```

Comment:
This currently prevents the site from building.

```suggestion
import HostNavbar from '@/components/HostNavbar';
```